### PR TITLE
OSX fixes

### PR DESCRIPTION
**Pull request by kumpera**
Fix compilation of imitation learning code and an unit test.

Fixes https://github.com/VowpalWabbit/reinforcement_learning/issues/138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,23 @@ vcpkg install flatbuffers:x64-windows
 
 You'll need to add the flatbuffers tool directory to your PATH aswell: `<vcpkg_root>\installed\x64-windows\tools\flatbuffers`
 
+## OSX
```
Comment (jackgerrits) [inline README.md:126]:
```suggestion
## MacOS
```

File: README.md [hunk 2]
```diff
@@ -123,6 +123,23 @@ vcpkg install flatbuffers:x64-windows
...
+
+MacOS dependencies can be managed through homebrew.
+
+```
+brew install cpprestsdk flatbuffers
+```
+
+In other to build using homebrew dependencies, you must invoke cmake this way:
```
Comment (jackgerrits) [inline README.md:134]:
```suggestion
In order to build using homebrew dependencies, you must invoke cmake this way:
```

File: README.md [hunk 3]
```diff
@@ -123,6 +123,23 @@ vcpkg install flatbuffers:x64-windows
...
+
+```
+mkdir build
+cd build
+cmake -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl@1.1 ..
```
Comment (jackgerrits) [inline README.md:139]: This is already in the readme

Comment (kumpera) [reply README.md:139]: So, this is not an optional step as of Catalina. :(

**Pull request merged**